src/user-parent.js

```diff
@@ -0,0 +1,47 @@
+class UserParent {
+  constructor(userData) {
+    this.users = userData;
+  }
+
```

Comment:
Hey, looks like all the test pass. But im not getting data on the dom. do we need to relink it?